batch7crud-roll3/src/main/java/com/lftechnology/batch7crud/service/StudentService.java

```diff
@@ -0,0 +1,45 @@
+package com.lftechnology.batch7crud.service;
+
+import java.util.ArrayList;
+import java.util.HashMap;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.HashMap'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)